Resolve conflicts from main

TalhaMaliktz opened the pull request.

fchavant commented on user-guide/lisk-core.md:
```suggestion
Upgrade an instance of Lisk Core (Binary) to a specified or latest version.
```

ManuGowda commented on user-guide/lisk-core.md:
The `core:upgrade` command is only supported for Lisk Core version `>v2.0.0`

fchavant commented on user-guide/lisk-core.md:
Don't we usually use version numbers (`2.0.0`) instead of tags (`v2.0.0`)?

fchavant commented on user-guide/lisk-core.md:
```suggestion
> To upgrade Lisk Core (Binary) `v1.6.0` or lower versions, please use the [bash script](../../lisk-core/setup/binary.md#option-b-the-bash-script), or [remove](#lisk-coreuninstall-name) the old version and make a fresh [install](#lisk-coreinstall-name).
```

Tschakki committed:
Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove line

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

Tschakki committed:
Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove line

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

Tschakki committed:
Release 05-07-2024 (#167)

Guide on Importing LSK to MetaMask
Add faucet URLs
Add exchanges page

Tschakki committed:
Release 15-07-2024 (#170)

### What was the problem?

This release includes

- #42 
- #154

Tschakki committed:
Release 23-07-2024 (#178)

### Inlcudes
 
- #174 
- #175 
- #176 
- #166

Tschakki committed:
Release 01-08-2024 (#183)

Includes:

- #157

Tschakki committed:
Release 05-08-2024 (#186)

Includes

- #173 
- #177

Tschakki committed:
Release 08.08.2024 (#189)

Includes:

- #180

Tschakki committed:
Release 28-08-2024 (#194)

### Includes 

- #187 
- #191

Tschakki committed:
Release 11-09-2024 (#207)

### Includes

- #99
- #181 
- #196
- #197  
- #204

Tschakki committed:
Release 19-09-2024 (#213)

### Includes

- #203 
- #205 
- #209

Tschakki committed:
Release 26-09-2024 (#219)

### Includes

- #202 
- #199

Tschakki committed:
Release 07-10-2024 (#221)

### Includes

- Add wallets page #217

Tschakki committed:
Release 15-10-2024 (#230)

### Includes

- #225

Tschakki committed:
Release 16-10-2024 (#231)

### Includes

- #182

TalhaMaliktz committed:
Release 12-11-2024 (#236)

[LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

TalhaMaliktz committed:
Release 2: 12-11-2024 (#238)

* [LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* [LISKDOCS-23] Update favicon (#237)

Update favicon

TalhaMaliktz committed:
Remove comment and coming soon

TalhaMaliktz committed:
Merge branch 'main' into Resolve-conflicts-from-main

TalhaMaliktz committed:
Merge branch 'development' into Resolve-conflicts-from-main
